cli/src/main/scala/filodb.cli/CsvImportExport.scala

```diff
@@ -20,8 +19,6 @@ import filodb.core._
 trait CsvImportExport {
   val system: ActorSystem
```

Comment:
shouldn't these vals be defs?

     trait Fu {
        def system: ActorSystem
     }

     class MyFu(val system: ActorSystem) extends Fu {

     }
     class MyOtherFu extends Fu {
         val system = ActorSystem("fu", ConfigFactory.load)
     }